docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -32,112 +32,229 @@ For example:
+    - name: Package OctoPetShopProductService
+      run: |
+        octo pack --id="OctoPetShop.ProductService" --format="Zip" --version="$PACKAGE_VERSION" --basePath="$GITHUB_WORKSPACE/artifacts/OctopusSamples.OctoPetShop.ProductService" --outFolder="$GITHUB_WORKSPACE/artifacts"
+    - name: Package OctoPetShopShoppingCartService
+      run: |
+        octo pack --id="OctoPetShop.ShoppingCartService" --format="Zip" --version="$PACKAGE_VERSION" --basePath="$GITHUB_WORKSPACE/artifacts/OctopusSamples.OctoPetshop.ShoppingCartService" --outFolder="$GITHUB_WORKSPACE/artifacts"
+```
 
-jobs:
+### Pushing artifacts to Octopus Server
+Once the artifacts are packaged, use the Octopus CLI Action to [push](https://octopus.com/docs/octopus-rest-api/octopus-cli/push) the packages to the Octopus Server built-in repository.  The following example pushes the packages created from the Package operation.
+
+```yaml
+    - name: Push OctoPetShop Database
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.Database.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+    - name: Push OctoPetShop Web
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.Web.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+    - name: Push OctoPetShop ProductService
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.ProductService.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+    - name: Push OctoPetShop ShoppingCartService
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.ShoppingCartService.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+```
 
-  build:
-    name: Build
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v2
-      
-      - name: Install Octopus CLI
-        uses: !include <image-version-install-octopus-cli-action>
-        with:
-          version: latest
-          
-      - name: list-octopusdeploy-deployments
-        run: octo list-deployments --server=${{ env.serverURL }} --apiKey=${{ secrets.apiKey }}
+### Creating a release
+Using the Octopus CLI Action, add a step to issue the [create-release](https://octopus.com/docs/octopus-rest-api/octopus-cli/create-release) command to create a release
+
+```yaml
+    - name: Create release
+      run: |
+        octo create-release --project="Octo Pet Shop" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
```

Comment:
We also have a create release action you can use instead of calling the CLI: https://github.com/marketplace/actions/create-release-in-octopus-deploy

https://octopus.com/blog/github-actions-for-octopus-deploy#creating-a-release-and-deploying-it-to-the-dev-environment

Reply:
Updated.